README.md

```diff
@@ -1,2 +1,86 @@
+
+## Note
+This is a web project, which will be heavily based on a GitHub application that will monitor all events in the organisation and award interaction points.
+
+
+# french
+
+Yotas est un système de récompense de l'engagement dans les projets open source. Les contributeurs open source de l'organisation sont récompensés pour les efforts et l'énergie qu'ils consacrent à la plateforme. Chaque interaction qu'ils ont sur GitHub leur donne un certain nombre de Yotas qui peuvent être échangés contre des goodies, du mentoring, des ebooks (etc...) dans une boutique OSS.
+
+## Pourquoi ?
+Contribuer à l'open source demande un certain effort, et nous aimerions que chaque contributeur soit récompensé pour le temps et l'énergie qu'il consacre à aider notre communauté.
+
+## Description
+Le yota sera une récompense liée à l'organisation, que l'utilisateur obtiendra en soumettant des issues, en faisant du code review, pull requests, etc. Ils pourront utiliser ces yotas bien gagnés pour acheter des choses dans notre boutique comme un tête à téte avec le lead du projet, des t-shirts, une connexion internet (nécessite de l'argent réel), du temps dans un espace de coworking (nécessite un partenariat) et bien plus encore.
+
+## Fonctionnement
+
+Yotas est un système qui permet aux organisations open source de décerner des prix à leurs contributeurs en fonction des ressources dont ils disposent.
+
+Voici le schéma :
+
+- Je suis une organisation X avec Y ressources 
+- Je veux redistribuer Y aux contributeurs actifs. 
+- Chaque fois qu'une personne contribue, elle gagne des yotas, et avec nos Y ressources nous mettons en place une boutique où le contributeur achète en yotas.
+- Une ressource peut être de l'argent ou n'importe quoi d'autre.
+
+**Notez que le but des yotas n'est pas de convertir la contribution en argent**.
+
+Ce n'est pas non plus un moyen de financer un projet open source, mais il peut permettre d'utiliser les fonds reçus pour récompenser les contributeurs.
+
+Les sponsors Github et les Yotas peuvent être utilisés ensemble comme suit :
+
+- L'organisation X reçoit un financement et dispose donc de Y ressources
+- Avec ces ressources, elle crée une boutique où elle place des articles que ses contributeurs peuvent acheter sur Yotas.
+
+Ce qui peut être mis dans une boutique sur Yotas peut être tout et n'importe quoi (mentoring, tutoriels, goodies,...) c'est à l'organisation de décider en fonction de ce qu'elle a. 
+
+Pour OSS, la première étape est d'offrir un mentorat aux juniors, et nous essayons d'avoir des issues abordables pour qu'ils puissent contribuer. Mais nous réfléchissons encore à ce que nous pouvons offrir d'autre, en particulier aux seniors.
+
+## Note
+Il s'agit d'un projet web, qui sera fortement basé sur une application GitHub qui suivra tous les événements de l'organisation et attribuera des points d'interaction.
+
+## License
+
+ [MIT](https://choosealicense.com/licenses/mit/)
```

Comment:
done

Reply:
@elhmn  This project is in their early stage and i suggest to put all necessary information in the README so that everyone could understand the idea

Then in the future, we can readapt the README as expected

Reply:
I agree 100% with you. 👍